pymchelper/readers/fluka.py

```diff
@@ -357,3 +361,38 @@ def get_unit_for_scoring(cls, scoring: str) -> str:
 
         # if unknown scoring, return empty string
         return ''
+
+
+@dataclass(frozen=True)
+class Axis:
+    """Single axis description"""
+
+    name: str
+    unit: str
+
+
+@dataclass(frozen=True)
+class AxesDescription:
+    """Axes descriptions"""
+
+    x: 'UsrbinAxes.Axis'
+    y: 'UsrbinAxes.Axis'
+    z: 'UsrbinAxes.Axis'
```

Comment:
Is it the correct way to specify the type ?
Shouldn't it rather be something like
`x : Axis` ?
or `field` method generator ?

Reply:
Good point. I initially used it as a nested class and forgot to update it afterward.